src/components/App.js

```diff
@@ -124,6 +100,9 @@ function App() {
     setItemsForDisplay(displayProducts)
   }, [items]);
 
+  useEffect(() => {
```

Comment:
I might be missing something, but what is this useEffect doing?

Reply:
Nothing. Removed